templates/blocks/old_covers.twig

```diff
@@ -25,7 +32,7 @@
 				<div class="row limit-visibility">
 					{% for cover in covers %}
 						<div class="col-lg-4 col-md-6 cover-card-column">
-							<div class="cover-card card-one" style="background-image: url({{ cover.image }});">
+							<div onclick="onCardClick('{{ cover.button_link }}')" class="cover-card card-one" style="background-image: url({{ cover.image }});">
```

Comment:
Do you think it's feasible to create the whole card as a `<a>` element and avoid javascript?
I'm thinking accessibility here, to makes sure the whole card is semantically marked as a link, since there will be no other link on it.

Reply:
I've tried this approach, but <a> tags can't have children... Then I've tried making an <a> tag positioned absolute and taking the whole div, but then the tags were no longer clickable and the rollover effect on the heading was also lost 🙁

Reply:
I've opened [this PR](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/293) to explore the `<a>` tag option further!